### DISCUSSION TITLE
셀카 모드 카메라 위치 세팅

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#28] [5. Q&A]
셀카 모드 기본으로 제공해주시는 클립은 찾아서 넣어봤는데 
카메라 위치를 어떤식으로 세팅할지가 난해합니다.

혹시 팁있으시면 공유 부탁드립니다 ㅠㅠ

**Comment by mentalcrash** [comment]
안녕하세요.
혹시 `ZepetoCharacterController` 에서 제공하는 기본 `ZepetoCamera` 를 사용 중이신가요?
기본 `ZepetoCamera`는 현재 시스템 상에서 제어가 되고 있기 때문에 임의로 제어하기가 적합하지 않습니다.

임의로 카메라를 제어하기 위해서는 카메라를 새로 만드시고 조작하시는걸 추천드립니다.
